compiler/rustc_passes/src/liveness.rs

```diff
@@ -962,10 +964,10 @@ impl<'a, 'tcx> Liveness<'a, 'tcx> {
 
                 // Now that we know the label we're going to,
                 // look it up in the continue loop nodes table
-                self.cont_ln
-                    .get(&sc)
-                    .cloned()
-                    .unwrap_or_else(|| span_bug!(expr.span, "continue to unknown label"))
+                self.cont_ln.get(&sc).cloned().unwrap_or_else(|| {
+                    self.ir.tcx.dcx().span_delayed_bug(expr.span, "continue to unknown label");
+                    self.ir.add_live_node(ErrNode)
+                })
```

Comment:
We can't (yet) know that we should not be doing liveness checks, because the error happens in check_mod_loops for which we have no path to forward a taint to here ("here" being `check_liveness`)

Reply:
fwiw this (at least used to?)  fire here I believe: https://github.com/rust-lang/rust/issues/113379

Reply:
Oh neat. More evidence that those ICEs I've caused are preexisting, but hard to trigger